crml/erc20-peg/contracts/ERC20Peg.sol

```diff
@@ -44,7 +44,7 @@ contract ERC20Peg is Ownable {
     // i.e. v[i], r[i], s[i] matches the i-th validator[i]
     function withdraw(address tokenType, uint256 amount, CENNZnetEventProof memory proof) payable external {
         require(withdrawalsActive, "withdrawals paused");
-        bytes32 message = abi.encodePacked(tokenType, amount, msg.sender);
+        bytes32 message = keccak256(abi.encodePacked(tokenType, amount, msg.sender));
```

Comment:
```suggestion
        bytes32 message = abi.encodePacked(tokenType, amount, msg.sender)
```

Reply:
leave the message hashing to the bridge contract

Reply:
<img width="959" alt="Screen Shot 2021-08-30 at 1 49 53 PM" src="https://user-images.githubusercontent.com/29415595/131274467-a3d44a0b-08ba-499a-ad80-3a9297219554.png">
If I don't use keccak256, then it fails to compile

Reply:
ah should be `bytes` not `bytes32`

Reply:
done... ✅ 
used 
`byte memory message`